### DISCUSSION TITLE
Why do the directories have no color in the bash through SSH?

### DISCUSSION THREAD

**Discussion by cpcopy** [microsoft/terminal#15800] [Q&A] [answered]
Login into a linux server through SSH. Run 'ls' command to list the files and directories. 
In windows terminal, the color of directories is black. I use Solarized Light color scheme.
![image](https://github.com/microsoft/terminal/assets/80554779/3e75be9a-1480-406d-8e69-7396fafe5715)

While in putty, it is blue.
![image](https://github.com/microsoft/terminal/assets/80554779/f6a5ac73-170a-408a-ba03-47b16780f771)
I think it shall be a configuration issue. I went through all the configurations and searched on google, but didn't find anything. What is the configuration that I missed? Thanks very much.

**Comment by zadjii-msft** [comment, marked as answer]
> I use Solarized Light color scheme.

### [DAYS SINCE I'VE REGRETTED SHIPPING SOLARIZED: ~~168~~ 0](https://github.com/microsoft/terminal/issues/14887#issuecomment-1437479490)

Solarized is just plain a bad terminal theme. You'll note we're using it [as defined](https://github.com/altercation/solarized#the-values), and yea, the bright colors are all pretty much just grayscale.

**Reply by cpcopy** [reply]
Thanks. Now I understand the reason. Using solarized light from [terminal.sexy ](https://github.com/stayradiated/terminal.sexy/tree/master) works good.
